ProfilingTools/src/ArmoniK.Core.ProfilingTools.csproj

```diff
@@ -36,4 +36,8 @@
     <ProjectReference Include="..\..\Adaptors\MongoDB\src\ArmoniK.Core.Adapters.MongoDB.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+	<AssemblyAttribute Include="System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverageAttribute" />
```

Comment:
Why is this (non-test) project is excluded from coverage?

Reply:
This is also a helper project to collect traces, I don't think its part of the core functionalities of the library so it should not be tested.